### ISSUE TITLE
Audit enabled views

### ISSUE THREAD

**Issue by mirie** [NYC-Camp/website#65]
We have a good number of Views enabled on admin/structure/views.

Questions:
- Are we using all of them?
- Where are the various displays being used?
- Which should be QA'ed/updated?

@ForestMars, @willykaram -- this is sort of question for you so that we can focus in on updating these bad boys and get on the same page on what's being used, where it's used, and for what purpose.

**Comment by jonpugh** [comment]
I doubt anyone "knows" the answers to these questions. 

I suggest trying https://drupal.org/project/views_usage_audit to help give an overview of what is what!

**Comment by ForestMars** [comment]
Let me just note that when the camp was finished last year, the site did not have a lot of cruft. Pretty much everything that was added was used. I just went through the views list and see that pretty much all the custom views you see were ones we specifically added and used. 

I should also not that while we didn't hack a single line of code in any COD module last year (which should really go without saying, and shouldn't even have to mention that at all!) I cannot guarantee we didn't extend some of the views that come with COD. In fact, I feel like I remember we had to, specifically in the case of the community displays? It's a bit blurred in the rear view at this point.
